config/country_codes.yml

```diff
@@ -1,254 +0,0 @@
-      "MY": Malaysia
-      "MV": Maldives
-      "ML": Mali
-      "MT": Malta
-      "MH": Marshall Islands
-      "MQ": Martinique
-      "MR": Mauritania
-      "MU": Mauritius
-      "YT": Mayotte
-      "MX": Mexico
-      "FM": Micronesia, Federated States of
-      "MD": Moldova, Republic of
-      "MC": Monaco
-      "MN": Mongolia
-      "ME": Montenegro
-      "MS": Montserrat
-      "MA": Morocco
-      "MZ": Mozambique
-      "MM": Myanmar
-      "NA": Namibia
-      "NR": Nauru
-      "NP": Nepal
-      "NL": Netherlands
-      "NC": New Caledonia
-      "NZ": New Zealand
-      "NI": Nicaragua
-      "NE": Niger
-      "NG": Nigeria
-      "NU": Niue
-      "NF": Norfolk Island
-      "MP": Northern Mariana Islands
-      "NO": Norway
-      "OM": Oman
-      "PK": Pakistan
-      "PW": Palau
-      "PS": Palestine, State of
-      "PA": Panama
-      "PG": Papua New Guinea
-      "PY": Paraguay
-      "PE": Peru
-      "PH": Philippines
-      "PN": Pitcairn
-      "PL": Poland
-      "PT": Portugal
-      "PR": Puerto Rico
-      "QA": Qatar
-      "RE": Réunion
-      "RO": Romania
-      "RU": Russian Federation
-      "RW": Rwanda
-      "BL": Saint Barthélemy
-      "SH": Saint Helena, Ascension and Tristan da Cunha
-      "KN": Saint Kitts and Nevis
-      "LC": Saint Lucia
-      "MF": Saint Martin (French part)
-      "PM": Saint Pierre and Miquelon
-      "VC": Saint Vincent and the Grenadines
-      "WS": Samoa
-      "SM": San Marino
-      "ST": Sao Tome and Principe
-      "SA": Saudi Arabia
-      "SN": Senegal
-      "RS": Serbia
-      "SC": Seychelles
-      "SL": Sierra Leone
-      "SG": Singapore
-      "SX": Sint Maarten (Dutch part)
-      "SK": Slovakia
-      "SI": Slovenia
-      "SB": Solomon Islands
-      "SO": Somalia
-      "ZA": South Africa
-      "GS": South Georgia and the South Sandwich Islands
-      "SS": South Sudan
-      "ES": Spain
-      "LK": Sri Lanka
-      "SD": Sudan
-      "SR": Suriname
-      "SJ": Svalbard and Jan Mayen
-      "SZ": Swaziland
-      "SE": Sweden
-      "CH": Switzerland
-      "SY": Syrian Arab Republic
-      "TW": Taiwan, Province of China
-      "TJ": Tajikistan
-      "TZ": Tanzania, United Republic of
-      "TH": Thailand
-      "TL": Timor-Leste
-      "TG": Togo
-      "TK": Tokelau
-      "TO": Tonga
-      "TT": Trinidad and Tobago
-      "TN": Tunisia
-      "TR": Turkey
-      "TM": Turkmenistan
-      "TC": Turks and Caicos Islands
-      "TV": Tuvalu
-      "UG": Uganda
-      "UA": Ukraine
-      "AE": United Arab Emirates
-      "GB": United Kingdom
-      "US": United States
-      "UM": United States Minor Outlying Islands
-      "UY": Uruguay
-      "UZ": Uzbekistan
-      "VU": Vanuatu
-      "VE": Venezuela, Bolivarian Republic of
-      "VN": Viet Nam
-      "VG": Virgin Islands, British
-      "VI": Virgin Islands, U.S.
-      "WF": Wallis and Futuna
-      "EH": Western Sahara
-      "YE": Yemen
-      "ZM": Zambia
-      "ZW": Zimbabwe
```

Comment:
We are going to handle certificate generation in an entirely different way, so we no longer need all the certificate setup stuff on velum.